README.md

```diff
@@ -97,32 +100,96 @@ Add True Myth to your dependencies:
     npm install true-myth
     ```
 
-For ES6-module-friendly consumers, you can import the modules directly, or import them from the root module:
+For ES6-module-friendly consumers, you can import the modules directly, or import them from the root module.
+
+So this works:
 
 ```typescript
-// this works:
 import Maybe from 'true-myth/maybe';
 import Result from 'true-myth/result';
+```
+
+And this also works:
 
-// this also works:
+```typescript
 import { Maybe, Result } from 'true-myth';
 ```
 
+For use in Node.js, see [below][node].
+
 In Node.js, the TypeScript-generated CommonJS package cannot be imported as nested modules, but the modules still can be imported directly from the top-level module:
 
 ```typescript
 const { Maybe, Result } = require('true-myth');
 ```
 
-The build includes both ES6 modules and CommonJS modules, so you may reference them directly from their installation in the `node_modules` directory. (This may be helpful for using the library in different contexts, with the ES modules being supplied especially so you can do tree-shaking with e.g. Rollup.)
+The build includes ES6 modules, so you may reference them directly from their installation in the `node_modules` directory. (This may be helpful for using the library in different contexts, with the ES modules being supplied especially so you can do tree-shaking with e.g. Rollup.)
```

Comment:
Need to rewrite given that it's *only* ES6 modules going forward and the old way is deprecated.